creatorkit/Toolbox.md

```diff
@@ -138,17 +134,29 @@ Initially you need to install dependencies required to build. run the following
 
 If you do not have repo installed run the following
 
+<code>$ mkdir -p ~/bin</code>
+
 <code>$ curl https://storage.googleapis.com/git-repo-downloads/repo > ~/bin/repo</code>
 
 <code>$ chmod a+x ~/bin/repo</code>
 
+Update ~/.bashrc to add repo path in linux path permanently
+
+<code>export PATH=$PATH:~/bin/repo</code>
 
-You should now be able to run the following in a directory of your choice.
+You should now be able to run the following in a directory of your choice to get the code.
 
-<code>$ repo init -u  https://github.com/CreatorKit/manifest.git</code>
+<code>$ repo init -u  https://github.com/CreatorKit/manifest.git -b dev</code>
 
 <code>$ repo sync</code>
 
+In order to build Contiki applications and flash them to the Clicker boards, you need to install MPLABX IPE and the XC32 compiler from [Microchip](http://microchip.wikidot.com/mplabx:installation). Note that installing the IDE/IPE will link you to the XC32 compiler install instructions once complete. You may need to reboot your machine after installing the compiler.
+
+You can now build everything with the following commands.
+
 <code>$ cd build</code>
 
 <code>$ make</code>
+
```

Comment:
might be worth mentioning for more details please refer https://github.com/CreatorKit/build/blob/dev/README.md